### ISSUE TITLE
Unable to Install/Find it in menu please..

### ISSUE THREAD

**Issue by lokkiii** [armory3d/armory#1941]
Hello Dev Team :)

**Description**
Thank you for making Armory3D, it looks promising.. But after I install. I think this error comes please.... (Installed Blender 2.83.7 LTS)

```
location: C:\Program FilTraceback (most recent call last):
  File "C:\Users\xxxx\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\armory.py", line 380, in on_load_post
    bpy.ops.arm_addon.start()
  File "C:\Program Files\Blender Foundation\Blender 2.83\2.83\scripts\modules\bpy\ops.py", line 201, in __call__
    ret = op_call(self.idname_py(), None, kw)
RuntimeError: Error: Traceback (most recent call last):
  File "C:\Users\xxxx\AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons\armory.py", line 259, in execute
    import start
  File "C:\ArmorySDK//armory/blender\start.py", line 1, in <module>
    import arm.nodes_logic
  File "C:\ArmorySDK//armory/blender\arm\nodes_logic.py", line 11, in <module>
    from arm.logicnode import arm_nodes
  File "C:\ArmorySDK//armory/blender\arm\logicnode\__init__.py", line 53, in <module>
    _module = importlib.import_module(f'{__name__}.{module_name}')
  File "C:\Program Files\Blender Foundation\Blender 2.83\2.83\python\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "C:\ArmorySDK//armory/blender\arm\logicnode\action_add_group.py", line 6, in <module>
    class AddGroupNode(Node, ArmLogicTreeNode):
TypeError: Cannot create a consistent method resolution
order (MRO) for bases Node, ArmLogicTreeNode

location: C:\Program Files\Blender Foundation\Blender 2.83\2.83\scripts\modules\bpy\ops.py:201

```

_I have downloaded the Armory3D SDK for the first time on the new computer/fresh Win10 installation..  (So no previous versions of blender/armory)_

 _I have extracted the SDK to the location "C:\ArmorySDK", so that there are no spaces in the path.. Also, in Blender>Addons tab, the "SDK path" is populated to the same folder above.._ 

**And, this seemed to have been resolved in the post below.  But it still occurs to me.. 
https://github.com/armory3d/armory/pull/1711 **

Are there any specific instructions for me to follow please...

**To Reproduce**
Newly installed.   After installation, Armory doesn't appear in the menu on the right.  Then when I restart it, there is still no armory in the menu.  There are errors when I try to remove it from addons, and install it back... 

**System**
Blender: 2.83.7 LTS
Armory: (2020, 10, 0)
OS: Win 10 Pro
Graphics card: GTX 1050Ti

**Test File**
*(drag & drop the zipped .blend file here)*

Below Screenshot is when I try to uninstall and reinstall from Addons menu in blender:

![ARMORY_ISSUE](https://user-images.githubusercontent.com/48851151/96453353-f6ea8d00-1237-11eb-920e-99639cbe0868.jpg)

**Comment by MoritzBrueckner** [comment]
Hi and welcome,

it looks like you somehow have a SDK version that's partly broken. The error message tells you that the file that created the error is in this path: `C:\ArmorySDK//armory/blender\arm\logicnode\action_add_group.py`, but since I think SDK 2020.09 the file was renamed to `LN_create_collection.py` and can now be found here: https://github.com/armory3d/armory/blob/master/blender/arm/logicnode/scene/LN_create_collection.py. So at least the `logicnode` folder in your Armory installation is broken.

From where did you download the SDK? I looked at the version from [itch.io](https://armory.itch.io/armory3d) and it looks good to me.

The error message on the right side of the screenshot is probably caused by the fact that you couldn't properly uninstall Armory because of the left side error.

**Comment by lokkiii** [comment]
Hi MoritzBrueckner :)

Thank you for your quick response.  I have actually downloaded the file from here. 

https://armory.itch.io/armory3d/download/eyJleHBpcmVzIjoxNjAzMTI2Mzk0LCJpZCI6OTI0MjN9%2e9meAvs9HB%2b3Sd1fcMTB7oE3%2f%2fxw%3d

It is a 124mb file.

![Armory_Download](https://user-images.githubusercontent.com/48851151/96485349-30cc8b00-125a-11eb-801b-7e8c790be4ad.jpg)


I have again downloaded it now, and did the following...
>> Removed it from blender Addons menu.  Close blender.
>> Deleted the ArmorySDK folder.  Unzipped the Newly downloaded file to "Armory"
>> Started Blender again, and then installed it from Addons menu.

Absolutely no difference though.  It just fails to appear..

So I have downloaded the single file at fault from the link you have provided previously, [action_add_group.py](https://github.com/armory3d/armory/blob/master/blender/arm/logicnode/scene/LN_create_collection.py).


Replaced the file in the directory as you have pointed in this location.
```
C:\ArmorySDK\armory\blender\arm\logicnode
```

Removed the addon from Blender and restarted and installed the addon again.  And no difference...

I am unsure what am I doing wrong..

**Comment by MoritzBrueckner** [comment]
Hmm, thats weird.

So you never installed any other Armory version other than the SDK 20.10, right? Did you move some Blender folders (installation/AppData/whatever) over to the new Windows version? Also, make sure that there is no SDK somewhere in those folders.

Can you check whether there are some `__pycache__` folders with `.pyc` files (cached python scripts) in the same folder Blender puts `armory.py` in (probably in `AppData\Roaming\Blender Foundation\Blender\2.83\scripts\addons`)? If yes, delete that after uninstalling and even better delete `armory.py` as well (if it still is in that folder) before trying another install.

If that all doesn't help: can you post a screenshot of the content of the `logicnode` folder that is inside the SDK? It's not only that single file mentioned above, the entire folder structure has changed in the last SDK versions.

**Comment by lokkiii** [comment]
Thank you so much @MoritzBrueckner ... You are right, I had copied some folders from the %appdata% folder from the previous Windows.   It were those addons .  But I am sure some Armory files may have been copied over;; and I just deleted the entire Blender folder inside "Roaming" and then repaired the Blender installation.  And now all the controls show perfectly well.  Thank you so much for pointing out to this..    And you guys are doing an amazing job with Armory... I am your biggest fan... Thank you for your time and patience again...
